test/timetree_calenar_app_client_test.rb

```diff
@@ -56,7 +56,7 @@ def test_initialize_without_private_key
   end
 
   #
-  # test for TimeTree::Client#calendar
+  # test for TimeTree::OAuthApp::Client#calendar
```

Comment:
This might be `TimeTree::CalendarApp::Client`.
I'm sorry my previous comment is also wrong.